editor/src/helpers/environment.js

```diff
@@ -1 +1,8 @@
 export const isDev = window.location.hostname === "localhost";
+
+/**
+ * Use this code to apply breaking changes to local storage settings.
+ * when the code name is changed, user's will have their local storage
+ * reset. the code name is the VZ release name
+ */
+export const appCodeName = "asclepias";
```

Comment:
I thought you were speaking in a secret code until I realized this was the VZ release name for 1.48